app/angular/src/client/preview/angular-beta/RenderNgAppService.ts

```diff
@@ -19,14 +21,72 @@ import { isComponentAlreadyDeclaredInModules } from './NgModulesAnalyzer';
 export class RenderNgAppService {
   private static instance: RenderNgAppService;
 
+  public static SELECTOR_STORYBOOK_WRAPPER = 'storybook-wrapper';
+
   public static getInstance() {
     if (!RenderNgAppService.instance) {
       RenderNgAppService.instance = new RenderNgAppService();
     }
     return RenderNgAppService.instance;
   }
 
-  public static SELECTOR_STORYBOOK_WRAPPER = 'storybook-wrapper';
+  public static getNgModuleMetadata = (
+    {
+      storyFnAngular,
+      parameters,
+    }: {
+      storyFnAngular: StoryFnAngularReturnType;
+      parameters: Parameters;
+    },
+    storyProps$: Subject<ICollection>
+  ): NgModule => {
+    const {
+      component: storyComponent,
+      props,
+      styles,
+      template,
+      moduleMetadata = {},
+    } = storyFnAngular;
+
+    if (storyComponent) {
+      logger.warn(dedent`\`component\` story input is deprecated, and will be removed in Storybook 7.0.
```

Comment:
Please use `util-deprecate`. That way it's easier to find all the instances before 7.0 and remove them.

Example: https://github.com/storybookjs/storybook/blob/master/lib/core/src/client/preview/loadCsf.ts#L9

Reply:
I thought, it was only for functions. It's done https://github.com/storybookjs/storybook/pull/13383/commits/6e6de1b357b28ef6cac13dc2ba7a09c731e8c11f#diff-57fd10a2b9f215006eeda934f6e927d27c8773e71e488b3c04d9a6168fe303c0R52